src/kvstore.h

```diff
@@ -8,7 +8,9 @@ typedef struct _kvstore kvstore;
 typedef struct _kvstoreIterator kvstoreIterator;
 typedef struct _kvstoreHashtableIterator kvstoreHashtableIterator;
 
+/* Return 1 if we should skip the hashtable in scan. */
 typedef int(kvstoreScanShouldSkipHashtable)(hashtable *d);
+/* Return 1 if we should skip the hashtable in expand. */
 typedef int(kvstoreExpandShouldSkipHashtableIndex)(int didx);
```

Comment:
So the isValid function is actually a shouldSkip callback? We should rename all to isValid IMO. (Or rename all to shouldSkip.)

Something like this?

```suggestion
typedef bool(kvstoreExpandHashtableIndexIsValid)(int didx);
```

Reply:
yes, it is actually a skip_cp, i updated it to use skip (seem better and less diff)